packages/components/forma-36-react-timepicker/src/Timepicker.tsx

```diff
@@ -79,7 +79,7 @@ function createHours() {
     hours.push(addMinutes(addHours(startOfDay(new Date()), hour), 30));
   }
   return orderBy(hours, (time: Date) => time, 'asc').map((m: Date) =>
-    format(m, DATEFNS_12H_FORMAT)
+    format(m, DATEFNS_12H_FORMAT),
```

Comment:
The changes in this file were done by prettier

`tsdx lint` was complaining about its formatting